### PR TITLE
support for kvp_client on azhpc-images

### PR DESCRIPTION
**Pull request by abhamidipati0614**
- kvp_client available at /opt/azurehpc/tools
- Minor bug fix for package update on AlmaLinux*

### CODE REVIEW

File: alma/alma-8.x/alma-8.6-hpc/install.sh [hunk 1]
```diff
@@ -61,6 +61,9 @@ $COMMON_DIR/install_health_checks.sh
 # copy test file
 $COMMON_DIR/copy_test_file.sh
 
+# copy kvp client file
```
Comment (jithinjosepkl) [inline alma/alma-8.x/alma-8.6-hpc/install.sh:64]: maybe move this into utils?

File: common/copy_kvp_client.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+DEST_DIR=/opt/azurehpc
```
Comment (jithinjosepkl) [inline common/copy_kvp_client.sh:4]: can we use /opt/azurehpc/tools  folder?

File: alma/alma-8.x/alma-8.6-hpc/install.sh (hunk 1 above)
Comment (abhamidipati0614) [reply alma/alma-8.x/alma-8.6-hpc/install.sh]: Moved to utils

File: common/copy_kvp_client.sh (hunk 2 above)
Comment (abhamidipati0614) [reply common/copy_kvp_client.sh]: Changed the dest dir

**Review by jithinjosepkl — APPROVED** [review]

**Pull request merged**